Changefeed/ChangefeedSample/ChangefeedSample.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.ChangeFeed;
+using Azure;
+using System.IO;
+using System.Text;
+
+namespace changefeedjd
+{
+    public static class ChangefeedSample
+    {
+        /* This program outputs events from a storage account's changefeed, which may be filtered. 
+         * The program iterates through the changefeed using a cursor, which will be saved in a container of the specified storage account. 
+         * Before starting the program, delete the cursor from previous runs from the storage account, if it exists
+         */
+        [FunctionName("Function1")]
+        public static async void Run([TimerTrigger("0 */1 * * * *")]TimerInfo myTimer, ILogger log)
+        {
+            string connectionString = "CONNECTION_STRING";
+            string containerName = "example";
+
+            //Predicate to filter events from Changefeed
+            Predicate<BlobChangeFeedEvent> containerCheck = (BlobChangeFeedEvent changeFeedEvent) => { return changeFeedEvent.Subject.Contains("/containers/" + containerName + "/"); };
+                        
+            BlobServiceClient blobServiceClient = new BlobServiceClient(connectionString);
+            BlobChangeFeedClient changeFeedClient = blobServiceClient.GetChangeFeedClient();
+            BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient("storingcursorcontainer");
+            BlobClient blobClient = new BlobClient(connectionString, "cursorcontainer","storingcursorblob");                      
```

Comment:
Should we be giving the user the ability to name their Container and blob that stores the cursor themselves? I think it's extremely unlikely that the customer would name their blob or container to the be the same thing, but its just so they know where the cursor will live. I guess punting more control to the customer

Reply:
I'd extract this to config or constants. BTW. let's use nouns for container and blob name. I.e. "cursorStorage" or so.